packages/web/src/components/molecules/CommonSettings/Delete.tsx

```diff
@@ -14,17 +11,20 @@ interface DeleteProps {
   description: string;
   label: string;
   redirectUrl: string;
-  useDeleteMutation: UseMutation<MutationDefinition<string, any, any, void>>;
+  onDelete: (id: string) => Promise<any>;
 }
 
-const Delete: FC<DeleteProps> = ({name, description, label, redirectUrl, useDeleteMutation}) => {
-  const {open} = useModal({
+const Delete: FC<DeleteProps> = ({name, description, label, redirectUrl, onDelete}) => {
+  const {open, close} = useModal({
     title: `Delete this ${label}`,
     width: 600,
     content: (
       <DeleteEntityModal
         defaultStackRoute={redirectUrl}
-        useDeleteMutation={useDeleteMutation}
+        onClose={() => {
```

Comment:
We could consider using `const {close} = useModal()` from within of `DeleteEntityModal`, but that's optional

Reply:
moved to `DeleteEntityModal `